### ISSUE TITLE
[Error: timeout of 3000ms exceeded] timeout: 3000

### ISSUE THREAD

**Issue by chossimbaone** [mauimauer/portify#112]
Hi.
I am trying to use portify 0.51. I've used the fix from #110 and #95. I do however still get the error output below.

Im running it with ubuntu 14.04 LTS, Also i tried with a windows machine same issue. I've also tried the fixed with copying the tracks to a different playlist, and still no luck.

Any ideas?

"ERROR OUTPUT"
portify 0.51
Starting without harmony
Option log level is not valid. Please refer to the README.
Google Login success
Spotify Login success
got 4 playlists
starting transfer...
initiated gmusic
created playlist Improbable Banjo(spotify:user:alecwilson:playlist:5VeEIQhs74fu7ZaGfzYFCL) on google music
fetch info about playlist in spotify
gotPlaylistLength()
gmusic:search2(), error
{ [Error: timeout of 3000ms exceeded] timeout: 3000 }
gmusic:search2(), parsing failed
double callback!
gmusic:search2(), error
{ [Error: timeout of 3000ms exceeded] timeout: 3000 }
gmusic:search2(), parsing failed
gmusic:search2(), error
{ [Error: timeout of 3000ms exceeded] timeout: 3000 }
gmusic:search2(), parsing failed
gmusic:search2(), error
{ [Error: timeout of 3000ms exceeded] timeout: 3000 }
gmusic:search2(), parsing failed
gmusic:search2(), error
{ [Error: timeout of 3000ms exceeded] timeout: 3000 }
gmusic:search2(), parsing failed
double callback!
double callback!
double callback!
double callback!

**Comment by chossimbaone** [comment]
No one else is having this issue?

**Comment by sonofjon** [comment]
Yeah, I get this too sometimes. It's sporadic. Happens sometimes, sometimes not. I'm thinking the Google play API has a rate limit that causes this error after you have processed a maximum number of songs. This is just speculation though. 

It also seems that the error occurs less frequently during certain times of the day.
